src/drive/web/modules/navigation/FileExplorer.jsx

```diff
@@ -89,6 +153,15 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
   fetchMoreFiles: (folderId, skip, limit) =>
     dispatch(fetchMoreFiles(folderId, skip, limit)),
   fetchFolder: folderId => dispatch(openFolder(folderId)),
+  fetchFolderFromTrash: folderId =>
```

Comment:
@Crash-- Could we use `props.onFolderOpen` instead, as defined [in the TrashContainer](https://github.com/cozy/cozy-drive/blob/fix/QuickView/src/drive/web/modules/trash/Container.jsx#L46) ?

Reply:
No, because I'm silly and the TrashContainer is further down the tree, so the prop doesn't exist there.